akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -168,6 +168,20 @@ private[projection] object R2dbcOffsetStore {
 
   final class RejectedEnvelope(message: String) extends IllegalStateException(message)
 
+  sealed trait IsAcceptedResult
+
+  object IsAcceptedResult {
+    case object Accepted extends IsAcceptedResult
+    case object Duplicate extends IsAcceptedResult
+    case object RejectedSeqNr extends IsAcceptedResult
+    case object RejectedBacktrackingSeqNr extends IsAcceptedResult
```

Comment:
I find it confusing that the trait is called `IsAcceptedResult`, but it can be a rejection as well. 

Maybe better to call is `Validation` instead? 

```scala
  object Validation {
    case object Accepted extends Validation
    case object Duplicate extends Validation
    case object RejectedSeqNr extends Validation
    case object RejectedBacktrackingSeqNr extends Validation
```

Reply:
alright, I'll follow up with some renaming in separate PR